compiler/rustc_resolve/src/macros.rs

```diff
@@ -1179,6 +1182,9 @@ impl<'ra, 'tcx> Resolver<'ra, 'tcx> {
         );
 
         if let Some(builtin_name) = ext.builtin_name {
+            if attr_ext.is_some() {
+                self.dcx().emit_err(errors::BuiltinMacroWithNonInvocationRules { span, ident });
```

Comment:
I don't think this is necessary, `#[rustc_builtin_macro]` is unstable, and any rules of such macros are ignored anyway.

Reply:
@petrochenkov Should I just silently set the attr_ext to `None` for a built-in macro, then?

Reply:
I've now done this, and deleted the error.